pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -1122,7 +1122,7 @@ private String getBrokerAddress() {
         return String.format("%s:%s", pulsar.getAdvertisedAddress(),
                 pulsar.getConfiguration().getWebServicePort().isPresent()
                         ? pulsar.getConfiguration().getWebServicePort().get()
-                        : pulsar.getConfiguration().getWebServicePortTls());
+                        : pulsar.getConfiguration().getWebServicePortTls().get());
```

Comment:
If the Optional has no value `get` will throw a RuntimeException.
What about using orElse(null).
This case won't probably happen but it is better to not risk

Reply:
The same pattern is used elsewhere. There is a check in Broker startup that if webServicePort is Optional.empty, then webServicePortTls must be set. It should be fine to have the code as is.

Reply:
makes sense